packages/react/src/number-field/decrement/NumberFieldDecrement.test.tsx

```diff
@@ -76,6 +76,28 @@ describe('<NumberField.Decrement />', () => {
     expect(input).to.have.value('0.235');
   });
 
+  it('only calls onValueChange once per decrement', async () => {
+    const handleValueChange = spy();
+    const { user } = await render(
+      <NumberField.Root onValueChange={handleValueChange}>
+        <NumberField.Decrement />
+        <NumberField.Input />
+      </NumberField.Root>,
+    );
+
+    const button = screen.getByRole('button');
+
+    await user.click(button);
+    await waitFor(() => {
+      expect(handleValueChange.callCount).to.equal(1);
+    });
```

Comment:
Do we need the waitFor here? It seems that this is supposed to run synchronously, so it would be simpler like this?

```suggestion
    expect(handleValueChange.callCount).to.equal(1);
```

We can take it from the angle of a user. If our test starts to become async, this would potentially be a breaking change, so we would want to know about it?